lib/graphql/analysis_error.rb

```diff
@@ -1,4 +1,12 @@
 module GraphQL
   class AnalysisError < GraphQL::ExecutionError
+    def initialize(message, ast_node = nil)
```

Comment:
can we "future-proof" this by accepting a named argument instead, eg 

```ruby 
def initialize(message, ast_node: nil)
```

? Our future selves will thank us if this API grows!